test/functional/feature_assumeutxo.py

```diff
@@ -237,6 +237,9 @@ def run_test(self):
         # will allow us to test n1's sync-to-tip on top of a snapshot.
         self.generate(n0, nblocks=100, sync_fun=self.no_op)
 
+        with n0.assert_debug_log(expected_msgs=["[snapshot] activation failed - work does not exceed active chainstate","[snapshot] activation failed - population failed"]):
```

Comment:
Thanks for the review! 

I think the timeout should be fine there is even a comment pointing out that we check the work twice because we want to fail fast
https://github.com/bitcoin/bitcoin/blob/master/src/validation.cpp#L5655-L5657